setup.py

```diff
@@ -36,7 +36,7 @@
         'future>=0.14.0; python_version<"3"',
         'backports.functools_lru_cache; python_version<"3.2"',
         'jedi>=0.17.0,<0.18.0',
-        'python-jsonrpc-server>=0.3.2',
+        'python-jsonrpc-server>=0.3.2,<0.4',
```

Comment:
Why not

```suggestion
        'python-jsonrpc-server~=0.3.2',
```

? I think that would show the intent of "version `0.3.2` or any later, but compatible version".